tests/e2e/performance/test_fio_benchmark.py

```diff
@@ -161,26 +168,25 @@ def test_fio_workload_simple(self, ripsaw, es, interface, io_pattern):
             'resources/crds/'
             'ripsaw_v1alpha1_ripsaw_crd.yaml'
         )
-        sc = 'ocs-storagecluster-ceph-rbd' if interface == 'CephBlockPool' else 'ocs-storagecluster-cephfs'
+        if interface == 'CephBlockPool':
+            sc = 'ocs-storagecluster-ceph-rbd'
```

Comment:
Please save 'ocs-storagecluster-ceph-rbd' in constants.py

Reply:
done